src/Components/CreateImageWizardV2/steps/Snapshot/Snapshot.tsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react';
+
+import {
+  Alert,
+  Button,
+  DatePicker,
+  Flex,
+  FormGroup,
+  Grid,
+  Radio,
+  Text,
+  Title,
+} from '@patternfly/react-core';
+
+import ConditionalTooltip from './components/ConditionalTooltip';
+
+import { useListFeaturesQuery } from '../../../../store/contentSourcesApi';
+import { useAppDispatch, useAppSelector } from '../../../../store/hooks';
+import {
+  selectSnapshotDate,
+  selectUseLatest,
+  changeUseLatest,
+  changeSnapshotDate,
+} from '../../../../store/wizardSlice';
+import {
+  dateToMMDDYYYY,
+  parseMMDDYYYYtoDate,
+} from '../../../../Utilities/time';
+
+const dateValidators = [
+  (date: Date) => {
+    if (date.getTime() > Date.now()) {
+      return 'Cannot set a date in the future';
+    }
+    return '';
+  },
+];
+
+export default function Snapshot() {
+  const dispatch = useAppDispatch();
+  const snapshotDate = useAppSelector(selectSnapshotDate);
+  const useLatest = useAppSelector(selectUseLatest);
+  const setUseLatest = (val: boolean) => dispatch(changeUseLatest(val));
+  const setSnapshotDate = (val: string) => dispatch(changeSnapshotDate(val));
+
+  // TODO: Remove query once snapshotting is no longer a "feature flag" i.e. when in Prod-stable
+  // Ensure to remove the Alert and ConditionalTooltip below.
+  const { data, isSuccess, isFetching, isError } = useListFeaturesQuery();
+  const isSnapshotDisabled =
+    isError ||
+    isFetching ||
+    (isSuccess &&
+      data?.snapshots?.accessible === false &&
+      data?.snapshots?.enabled === false);
+
+  return (
+    <>
+      <FormGroup>
+        <Radio
+          id="use latest snapshot radio"
+          ouiaId="use-latest-snapshot-radio"
+          name="use-latest-snapshot"
+          label="Use latest content"
+          description="Use the newest repository state available when building this image."
+          isChecked={useLatest}
+          onChange={() => !useLatest && setUseLatest(true)}
+        />
+        <ConditionalTooltip
+          show={isSnapshotDisabled}
+          setDisabled
+          position="left"
+          content="Snapshotting is not currently available"
+        >
+          <Radio
+            id="use snapshot date radio"
+            ouiaId="use-snapshot-date-radio"
+            name="use-snapshot-date"
+            label="Use a snapshot"
+            description="Target a date and build images with repository information from this date."
+            isChecked={!useLatest}
+            onChange={() => useLatest && setUseLatest(false)}
+          />
+        </ConditionalTooltip>
+        {isError && (
```

Comment:
Would we also remove this when snapshotting goes into prod-stable?

Reply:
Indeed